lib/buffer.js

```diff
@@ -59,6 +60,25 @@ Buffer.prototype.swap32 = function swap32() {
   return swap32n.apply(this);
 };
 
+Buffer.prototype.swap64 = function swap64() {
```

Comment:
Since we're here can we move all three down to where other prototype methods are? They're out of place here at the top of the file. Separate commit or two different commits, doesn't matter to me.